Richard Reis & Ashlei Jones - Testing

richardreeze opened the pull request.

richardreeze committed:
First commit

richardreeze committed:
Finished basic JavaScript step 1

poindextress committed:
adding the test functions for project-2

also fixing linter errors for project-1

poindextress committed:
adding the test functions for project-2

also fixing linter errors for project-1

poindextress committed:
removing yarn.lock

poindextress committed:
Merge branch 'master' of https://github.com/richardreeze/Testing

poindextress committed:
Merge branch 'master' of https://github.com/richardreeze/Testing

poindextress committed:
Merge branch 'master' of https://github.com/richardreeze/Testing

updating

ashleijo commented:
lol, sorry about all the commits. still learning how to rm pushed files properly. :)

mixelpixel commented:
That's all good @ashleijo 👍  - I say the more commit messages the merrier!

poindextress committed:
adding some changes to advanced-javascript

changing the expects to be functions in the arrays.test

adding a test for keys to object.test

poindextress committed:
actually adding the keys function to object.tests this time

richardreeze committed:
Finish basic JS project 1 test

mixelpixel commented:
ohai!
